scripts/mail-all-activity.py

```diff
@@ -1,181 +0,0 @@
-#!/usr/bin/env python
```

Comment:
`mail-all-activity.py` is being merged with `report_all_instructor_activity.py` command into `instructors_activity` command.

Commands are run outside of HTTP request (with `./manage.py instructors_activity`), so there are no timeouts. The downside is that if we want to use Django mailing system, we need to supply some settings via envvars.